dnngior/fasta2model_CLI.py

```diff
@@ -0,0 +1,249 @@
+    return base_model
+
+def build_gapfilled_model_from_fasta(args):
+    from cobra.io import write_sbml_model
+    args.path_to_base_model = os.path.join(args.output_folder,'base_models',f'base_{args.model_name}{args.suffix_model}')
+    if not os.path.isfile(args.path_to_base_model):
+        base_model = build_base_model(args)
+        write_sbml_model(cobra_model = base_model, filename = args.path_to_base_model)
+    else:
+        logging.warning(f'# Base model {args.model_name} allready exists, skipping')
+    gapfill_model_wrapper(args)
+
+def gapfill_model_wrapper(args):
+    from cobra.io import write_sbml_model
+    from dnngior.gapfill_class import Gapfill
+    from dnngior.reaction_class import Reaction
+
+    args.path_to_gf_model = os.path.join(args.output_folder,'gapfilled_models',f'gf_{args.model_name}{args.suffix_model}')
+    if not os.path.isfile(args.path_to_gf_model):
+        if args.medium:
+            medium_name = os.path.basename(args.medium)
+            gf_model = Gapfill(args.path_to_base_model, medium_file = args.medium)
+        else:
+            medium_name = 'Complete'
+            gf_model = Gapfill(args.path_to_base_model)
+        write_sbml_model(cobra_model = gf_model.gapfilledModel, filename =  args.path_to_gf_model)
+        n_dr = len(gf_model.draft_reaction_ids)
+        n_gr = len(gf_model.added_reactions)
+        args.gf_data_file.write(f'{args.model_name}\t{n_dr}\t{n_gr}\t{n_dr+n_gr}\t{medium_name}\n')
+    else:
+        logging.warning(f'# Gapfilled model {args.model_name} allready exists, skipping')
+        # gf_model = Gapfill(draftModel=args.path_to_gf_model, gapfill=False)
+        # gf_model.gapfilledModel = gf_model.draftModel
+
+
+def create_output_folder(args):
+    if os.path.exists(os.path.dirname(args.output_folder)):
+        if args.fasta_folder:
+            base_model_folder = os.path.join(args.output_folder, 'base_models')
+            if not os.path.exists(base_model_folder):
+                os.makedirs(base_model_folder, exist_ok=True)
+            else:
+                print(f'# WARNING: base models folder allready exists ({base_model_folder})')
+
+        gf_model_folder = os.path.join(args.output_folder, 'gapfilled_models')
+        if not os.path.exists(gf_model_folder):
+            print('# Creating output_folder for gapfilled models')
+            os.makedirs(gf_model_folder, exist_ok=True)
+        else:
+            print(f'# WARNING: gapfilled models folder allready exists ({gf_model_folder})')
+        # extra_folder = os.path.join(args.output_folder, 'tmp')
+        # if not os.path.exists(extra_folder):
+        #     print('# Creating gapfill info folder')
+        #     os.makedirs(extra_folder, exist_ok=True)
+        # else:
+        #     print('# WARNING: tmp folder already exists')
+    else:
+        sys.exit('# ERROR: your output_folder should have parents')
+
+def main():
```

Comment:
Since this is your main function for the CLI, I suggest you add some documentation